styles/mandar.css

```diff
@@ -653,7 +653,7 @@ img {
 }
 
 .starwarsffg.sheet .sheet-body.shipattachment {
-  height: calc(100% - 22.75rem);
+  height: calc(100% - 24rem);

```

Comment:
"didn't I tell you this would be a pain to support"

yes, yes you did. I don't want to hear it. :)